packages/autoprofile/precompile/build-all-addons.js

```diff
@@ -8,7 +8,6 @@
 
 const os = require('os');
 const path = require('path');
-const { copyFileSync, existsSync, mkdirSync } = require('fs');
```

Comment:
Apparently we are not running eslint on this folder. I think we should, so that we catch stuff like this directly -> https://github.com/instana/nodejs/pull/723